stats.py

```diff
@@ -1,4 +1,6 @@
-import os, redis
+import os
+import redis
```

Comment:
This is minor, but all standard imports should come before third-party imports as per the [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) standard.

Reply:
Updated (previously imported package has been removed now).